website/static/js/quickProjectSearchPlugin.js

```diff
@@ -0,0 +1,478 @@
+                }},
+                    m('i.fa.fa-angle-down'));
+            }
+        }
+
+        function sortDateAsc(){
+            if (ctrl.loadingComplete()){
+                 return m('button', {'class': ctrl.colorSortButtons('dateAsc'), onclick: function() {
+                     ctrl.sortBySortState(ctrl.sortState('dateAsc'));
+                 }},
+                     m('i.fa.fa-angle-up'));
+            }
+        }
+
+        function sortDateDesc(){
+            if (ctrl.loadingComplete()){
+                return m('button', {'class': ctrl.colorSortButtons('dateDesc'), onclick: function() {
+                    ctrl.sortBySortState(ctrl.sortState('dateDesc'));
+               }},
+                    m('i.fa.fa-angle-down'));
+            }
+        }
+
+        // Sort button for xs screen
+        function ascending() {
+            if (ctrl.loadingComplete()){
+                return m('button', {'class': ctrl.colorSortButtonsXS('Asc'), onclick: function() {
+                     ctrl.directionSort('Asc');
+                     ctrl.sortDirectionGivenField();
+                }},
+                     m('i.fa.fa-angle-up'));
+            }
+        }
+
+        // Sort button for xs screen
+        function descending() {
+            if (ctrl.loadingComplete()){
+                return m('button', {'class': ctrl.colorSortButtonsXS('Desc'), onclick: function() {
+                    ctrl.directionSort('Desc');
+                    ctrl.sortDirectionGivenField();
+                }},
+                     m('i.fa.fa-angle-down'));
+            }
+        }
+
+        // Dropdown for XS screen - if sort on title on large screen, when resize to xs, 'title' is default selected
+        function defaultSelected() {
+            var selected = ctrl.preSelectField();
+            if (selected === 'alpha') {
+                return [m('option', {value: 'alpha', selected:'selected'}, 'Title'), m('option', {value: 'date'}, 'Modified')];
+            }
+            else {
+                return [m('option', {value: 'alpha'}, 'Title'), m('option', {value: 'date', selected:'selected'}, 'Modified')];
+            }
+        }
+
+        function searchBar() {
+            if (ctrl.loadingComplete()){
+                return m('div.m-v-sm.quick-search-input', [
+                    m('input[type=search]', {'id': 'searchQuery', 'class': 'form-control', placeholder: 'Quick search projects', onkeyup: function(search) {
+                        ctrl.filter(search.target.value);
+                        ctrl.quickSearch();
+                    }}),
+                    m('span', {onclick: function() {
+                        ctrl.filter('');
+                        document.getElementById('searchQuery').value = '';
+                        ctrl.quickSearch();
+                    }},  m('button', m('i.fa.fa-times')))
+                ]);
+            }
+        }
+
+        function displayNodes() {
```

Comment:
@sloria, what do I do about 
```
m('.row',
       [m('.col-sm-4.col-md-5.p-v-xs', m('.quick-search-col', project.attributes.title)),
        m('.col-sm-4.col-md-4.p-v-xs', m('.quick-search-col', $osf.contribNameFormat(project, numContributors, ctrl.getFamilyName))),
        m('.col-sm-4.col-md-3.p-v-xs', m('.quick-search-col', ctrl.formatDate(project)))]
));
```

This was inside the `displayNodes` function. Specifically, `ctrl.formatDate(project)`.    args.formatDate(), but what about the project?